### PR TITLE
Move solution pages under solutions to match the Solutions nav tab

### PR DESCRIPTION
**Pull request by raimbekovm**
## summary

the Solutions nav tab listed 17 pages that physically live in `docs/en/guides/`, so clicking a solution landed on a `/guides/...` URL that contradicted the tab. this moves those pages into `docs/en/solutions/` as git renames, repoints every affected relative link and nav entry, and updates the absolute docs URLs in `examples/` to the new paths.

redirects for the old `/guides/...` URLs are handled outside this repository and land alongside this change, so existing inbound links and indexed URLs keep resolving — `mkdocs.yml` has no redirect mechanism and its build output does not serve production (`.github/workflows/docs.yml` only triggers the docs deploy), so redirects added here would have no effect.

verified by resolving every relative `.md` link in `docs/en/` (0 broken / 3609, unchanged from before the move) and every `mkdocs.yml` nav entry (18 `solutions/` + 45 `guides/`, 0 missing).

Deleted: the 17 pages' old `docs/en/guides/` locations. pure relocation — 76 insertions / 76 deletions, net zero, no content added.

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://www.ultralytics.com/actions)</sub>

### 🌟 Summary
Moved 17 solution documentation pages from `docs/en/guides/` to `docs/en/solutions/` and updated references so the Solutions navigation uses matching URLs.

### 📊 Key Changes
- Renamed the 17 solution pages, including Analytics, Heatmaps, Object Counting, Speed Estimation, and Queue Management, under `docs/en/solutions/` without adding content.
- Updated the Solutions navigation in `mkdocs.yml` and the Solutions index links to use the new paths.
- Repointed affected relative links across the documentation and updated solution URLs in `examples/README.md`, the Region Counter README, and Heatmaps and Object Counting notebooks.
- Corrected links within moved pages to reference other solution pages and related guides from their new directory.
- The PR context reports that all 3,609 relative documentation links and 63 navigation entries resolve successfully.

### 🎯 Purpose & Impact
- Solution pages now appear at `/solutions/...`, matching the Solutions navigation tab and related documentation links.
- Existing `/guides/...` inbound URLs depend on redirects managed outside this repository; the page content and solution workflows are otherwise unchanged.

### CODE REVIEW

**Comment by UltralyticsAssistant** [comment]
👋 Hello @raimbekovm, thank you for submitting a `ultralytics/ultralytics` 🚀 PR! To ensure a seamless integration of your documentation updates, please review the following checklist:
- ✅ **Define a Purpose**: Clearly explain the purpose of your fix or feature in your PR description, and link to any [relevant issues](https://github.com/ultralytics/ultralytics/issues). Ensure your commit messages are clear, concise, and adhere to the project's conventions.
- ✅ **Synchronize with Source**: Confirm your PR is synchronized with the `ultralytics/ultralytics` `main` branch. If it's behind, update it by clicking the 'Update branch' button or by running `git pull` and `git merge main` locally.
- ✅ **Ensure CI Checks Pass**: Verify all Ultralytics [Continuous Integration (CI)](https://docs.ultralytics.com/help/CI) checks are passing. If any checks fail, please address the issues.
- ✅ **Update Documentation**: Update the relevant [documentation](https://docs.ultralytics.com/) for any new or modified features.
- ✅ **Add Tests**: If applicable, include or update tests to cover your changes, and confirm that all tests are passing.
- ✅ **Sign the CLA**: Please ensure you have signed our [Contributor License Agreement](https://docs.ultralytics.com/help/CLA) if this is your first Ultralytics PR by writing "I have read the CLA Document and I sign the CLA" in a new message.
- ✅ **Minimize Changes**: Limit your changes to the **minimum** necessary for your bug fix or feature addition. _"It is not daily increase but daily decrease, hack away the unessential. The closer to the source, the less wastage there is."_ — Bruce Lee

This is an automated welcome message, and an engineer will assist with the review. For more guidance, please refer to our [Contributing Guide](https://docs.ultralytics.com/help/contributing). Don't hesitate to leave a comment if you have any questions. Thank you for contributing to Ultralytics! 🚀

File: mkdocs.yml [hunk 1]
```diff
@@ -501,23 +501,23 @@ nav:
...
-      - Object Cropping: guides/object-cropping.md
-      - Parking Management: guides/parking-management.md
-      - Queue Management: guides/queue-management.md
-      - Security Alarm System: guides/security-alarm-system.md
-      - Similarity Search: guides/similarity-search.md
-      - Speed Estimation: guides/speed-estimation.md
-      - Track Objects in Zone: guides/trackzone.md
-      - VisionEye View Objects Mapping: guides/vision-eye.md
-      - Workouts Monitoring: guides/workouts-monitoring.md
+      - Analytics: solutions/analytics.md
```
Comment (UltralyticsAssistant) [inline mkdocs.yml:525]: 💡 **MEDIUM**: Moving these source files removes the previously published `/guides/...` routes. This config has no redirect plugin or redirect mapping, so existing external links, bookmarks, and indexed URLs such as `/guides/analytics/` will become 404s after deployment. Preserve redirects for all renamed guide URLs (or retain compatibility pages) before switching the nav paths.

**Review by UltralyticsAssistant — COMMENTED** [review]
## 🔍 PR Review

<sub>Made with ❤️ by [Ultralytics Actions](https://www.ultralytics.com/actions)</sub>

The internal link and navigation updates are consistent with the new Solutions directory. One compatibility issue remains: the renamed documentation routes need redirects to avoid breaking existing published `/guides/` URLs.

💬 Posted 1 inline comment

**Review by glenn-jocher — APPROVED** [review]
LGTM at 98c73721c8786fc7061165041a96594b5752bb55. The patch is a net-zero relocation: it deletes the 17 stale `guides/` owners, moves the pages under `solutions/`, and updates only their consumers. No duplicate compatibility pages or symptom guards were added; formatting and path checks pass. Per instruction, this is approved but not merged.

**Comment by glenn-jocher** [comment]
@raimbekovm not sure about this, maybe these shouldn't even be called solutions as the slug is already used in www.ultralytics.com/solutions

**Comment by raimbekovm** [comment]
@glenn-jocher I looked at how other libraries name this tier and how they handle the same www-vs-docs overlap.

| Name                            | Who uses it                                                                                       | Slug                                                   |
| ------------------------------- | ------------------------------------------------------------------------------------------------- | ------------------------------------------------------ |
| **Solutions**                   | MediaPipe (umbrella over Tasks / Model Maker / Studio)                                            | `developers.google.com/edge/mediapipe/solutions/`      |
| **Solutions + qualifier**       | AWS "Solutions Library", Databricks "Solution Accelerators", Google Cloud "Jump Start Solutions"  | `/solutions/accelerators`                              |
| **Cookbook / Recipes**          | OpenAI, Anthropic, PyTorch ("bite-size, ready-to-deploy examples"), Keras ("Quick Keras Recipes") | `cookbook.openai.com`, `/tutorials/recipes_index.html` |
| **Examples / Templates**        | Keras, scikit-learn, Cloudflare Workers, Vercel                                                   | `/examples`, `/templates`                              |
| **Reference apps / Blueprints** | NVIDIA DeepStream, NVIDIA Blueprints ("customizable reference applications")                      | `build.nvidia.com/blueprints`                          |

MediaPipe looks like our closest match — same domain, same shape (prebuilt pipelines over models) — and I think it's telling that they use "Solutions" as the umbrella rather than avoiding it.

On the marketing-vs-docs overlap, I couldn't find anyone who solved it by giving up the word:

- **GitLab** runs both — `about.gitlab.com/solutions/` and `docs.gitlab.com/solutions/` — disambiguated editorially, not by renaming.
- **Databricks** nests the technical notebooks _inside_ the marketing tree (`/solutions/accelerators`), using a qualifier noun.
- **AWS** merged them on purpose: one Solutions Library, explicitly "for business and technical use cases".
- **Snowflake** is the only mover ("Solutions Center" → "Developer Guides"), but it retired its marketing `/solutions` index around the same time with no stated reason, so I don't think there's a collision motive to read into it.

Worth noting that `developers.google.com/solutions` (Gemini) and `developers.google.com/edge/mediapipe/solutions/` are unrelated uses of the word on the _same_ host — so even Google isn't a clean split-by-host example.

Two options if we do rename:

1. **Keep "Solutions"**
2. **Solutions + qualifier** ("Solution Recipes", "Solution Guides") — the Databricks pattern; keeps the word and the `/solutions/` slug, changes only the nav label.

My preference would be to keep the API surface as-is and just move the docs pages. I think the name has real external gravity now — a lot of third-party tutorial and course material teaches YOLO solutions by that name — and I'd rather not break that for no functional gain. Happy to go whichever way you prefer.
